superset-frontend/plugins/plugin-chart-echarts/src/Timeseries/Area/index.ts

```diff
@@ -63,7 +55,7 @@ export default class EchartsAreaChartPlugin extends ChartPlugin<
         category: t('Evolution'),
         credits: ['https://echarts.apache.org'],
         description: t(
-          'Time-series Area chart are similar to line chart in that they represent variables with the same scale, but area charts stack the metrics on top of each other. An area chart in Superset can be stream, stack, or expand.',
+          'Area charts are similar to line charts in that they represent variables with the same scale, but area charts stack the metrics on top of each other.',
```

Comment:
We probably should change that description only if FF is enabled, like in other charts